tests/robot-tests/tests/libs/charts.robot

```diff
@@ -157,3 +157,51 @@ user checks map chart indicator tile contains
 user checks infographic chart contains alt
     [Arguments]  ${locator}  ${text}
     user waits until parent contains element  ${locator}  css:img[alt="${text}"]
+
+user configures basic horizontal bar chart
```

Comment:
I'd merge this keyword with `user configures basic vertical bar chart` and `user configures basic geographic chart` by including the chart type in the arguments, so it could be called like `user configures basic chart ${DATABLOCK_NAME} Horizontal bar 500 800`